### PR TITLE
Update pihole docs to describe services

### PR DESCRIPTION
**Pull request by johnluetke**
**Description:** Updates documentation to describe `pi_hole.enable` and `pi_hole.disable` 

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27055

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Thanks, @johnluetke! 👍

File: source/_components/pi_hole.markdown [hunk 1]
```diff
@@ -59,4 +64,24 @@ pi_hole:
...
+| ---------------------- | -------- | -------- | ----------- |
+| `duration` | `True` | timedelta | Time for which Pi-hole should be disabled | 
+
+_Note: This service requires `api_key` to be specified in the configuration._
+
+### Service `pi.hole_enable`
+
+Enable your Pi-hole.
+
+_Note: This service requires `api_key` to be specified in the configuration._
```
Comment (klaasnicolaas) [inline source/_components/pi_hole.markdown:85]: @frenck ehm note? not in a div?

Comment (frenck) [reply source/_components/pi_hole.markdown:85]: Yes/no... if felt like adding big note boxes here would be overkill tbh

Comment (johnluetke) [reply source/_components/pi_hole.markdown:85]: I opted not to use a colored box as well because its an optional configuration setting.

Comment (klaasnicolaas) [reply source/_components/pi_hole.markdown:85]: Alright no problem just a question 😉

**Pull request merged**